packages/styled-react/src/components/TextInput.tsx

```diff
@@ -0,0 +1,35 @@
+import {
```

Comment:
borrowed from https://github.com/primer/react/pull/6925 to get tests passing, I expect to merge that PR before this one so reviewers can ignore this since it'll be looked at in the context of the TextInput PR